test_utils/venv.py

```diff
@@ -18,6 +18,8 @@ def run_in_virtualenv(shell_script):
         venv_path = str(Path(parent_dir) / 'venv')
         try:
             subprocess.check_call(['virtualenv', '-p', sys.executable, '--clear', venv_path])
+            # TODO: Remove the following line when both the latest versions of pip and pip-tools work fine together
+            subprocess.check_call(f'. {venv_path}/bin/activate; pip install pip==20.0.2', env={}, shell=True)
```

Comment:
If there's a version constraint on pip here, shouldn't it also be in `pip.txt`?  And just use that file here?

Reply:
this is setting up a temporary directory and creating a virtual environment, it doesn't have access to the repo's requirement files, should we copy over the `pip.txt`?

Reply:
Yeah, I think we should probably use the same pip version in these testing virtualenvs that we use when generating repos (and in the generated repos).